cmd/burrow/commands/configure.go

```diff
@@ -313,42 +308,36 @@ func Configure(output Output) func(cmd *cli.Cmd) {
 				output.Fatalf("could not update burrow config: %v", err)
```

Comment:
You should remove `configOpts.configure` on line 306 above - I think we can do it up front